cypress/e2e/spec.cy.ts

```diff
@@ -108,33 +116,140 @@ describe("testbox script", () => {
         postMessage(${fakeLoginMessageStringified});`);
     });
 
-    cy.get("@loginHandler").should("have.been.calledOnceWith", fakeLoginMessage().testbox.data);
+    cy.get("@loginHandlerSpy").should("have.been.calledOnceWith", fakeLoginMessage().testbox.data);
   });
 
-  it("Login event waiting for more than 10 seconds", () => {
+  it("Login handler should return redirect url", () => {
```

Comment:
The aim of our test is to ensure that `Navigate handler is called for login handler return url`.
Making sure the login handler returns a URL would be outside of SDK scope and into partner implementation territory.

In general the test makes sense, would just remove the check for the value that is returned by the login handler and rename it.

Reply:
Yes, we want to test for the redirect being done to the right place, but no need to test the value that is returned by the handler, specially as this is literally mocked by the test itself.